app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -24,30 +30,58 @@
     </div>
     <div v-if="delegation.loaded && undelegatedValidators.length > 0">
       <h3 class="tab-header">
-        Inactive Delegations
+        Unbonding validators
         <i v-tooltip.top="unbondInfo" class="material-icons info-button"
           >info_outline</i
         >
       </h3>
       <table-validators :validators="undelegatedValidators" />
     </div>
+    <div v-if="delegation.loaded && undelegatedValidators.length > 0">
+      <h3 class="tab-header transactions">
+        Unbonding transactions
+        <i
+          v-tooltip.top="unbondTransactions"
+          class="material-icons info-button"
+        >
+          info_outline
+        </i>
+      </h3>
+      <template v-for="transaction in unbondingTransactions">
+        <tm-li-stake-transaction
+          :transaction="transaction"
+          :validators="undelegatedValidators"
+          :bonding-denom="bondingDenom"
+          :key="transaction.hash"
+        />
+      </template>
+    </div>
   </div>
 </template>
 
 <script>
-import { mapGetters } from "vuex"
-import { TmDataMsg, TmDataLoading } from "@tendermint/ui"
+import { mapGetters, mapState } from "vuex"
+import { TmDataMsg, TmDataLoading, TmLiStakeTransaction } from "@tendermint/ui"
 import TableValidators from "staking/TableValidators"
 import TmDataConnecting from "common/TmDataConnecting"
+import moment from "moment"
 
 export default {
   name: `tab-my-delegations`,
-  components: { TableValidators, TmDataMsg, TmDataConnecting, TmDataLoading },
+  components: {
+    TableValidators,
+    TmDataMsg,
+    TmDataConnecting,
+    TmDataLoading,
+    TmLiStakeTransaction
+  },
   data: () => ({
     bondInfo: `Validators you are currently bonded to`,
-    unbondInfo: `Your bonded validators in unbonding process`
+    unbondInfo: `Your bonded validators in unbonding process`,
+    unbondTransactions: `The transactions currently in unbouding period`
```

Comment:
```suggestion
    unbondTransactions: `The transactions currently in unbonding period`
```

Reply:
#no-bonding ;)

Reply:
that's how the term is called, unbonding period. We agreed not to use bonding instead of delegation